demo/grid/fullsize-page-with-card.html

```diff
@@ -61,44 +61,43 @@
       },
 
       sharedElements: {
-        type: Object,
-        value: function() {
-          return {
-            'hero': this.$.card,
-            'ripple': this.$.fixed
-          }
-        }
+        type: Object
       },
 
       animationConfig: {
-        type: Object,
-        value: function() {
-          return {
-            'entry': [{
-              name: 'ripple-animation',
-              id: 'ripple',
-              toPage: this,
-            }, {
-              name: 'hero-animation',
-              id: 'hero',
-              toPage: this,
-              timing: {
-                delay: 150
-              }
-            }],
-            'exit': [{
-              name: 'fade-out-animation',
-              node: this.$.fixed
-            }, {
-              name: 'transform-animation',
-              transformFrom: 'none',
-              transformTo: 'translate(0px,-200vh) scale(0.9,1)',
-              node: this.$.card
-            }]
-          }
-        }
+        type: Object
       }
+    },
 
+    attached: function() {
+      this.sharedElements = {
```

Comment:
this is different from 1.x, as at every `attached` you'd override `this.animationConfig` and `this.sharedElements`.
WDYT of just having a check before this line like `if (this.animationConfig) return;`?